src/sentry/auth/access.py

```diff
@@ -277,7 +317,7 @@ def __init__(
         )
 
         super().__init__(
-            member=member,
+            _member=member,
```

Comment:
Should the kwarg have a leading `_` as well?

Reply:
In this case, no.   The interface for this existing Access implementation does in fact take a member, *but the resulting public interface does not intend to expose it publicly*.

In a follow up PR, I imagine @RyanSkonnord is going to implement a super class abstract base interface (the public interface), and underneath that start implementing a chain of access that works purely off the organization service data class objects.